pages/price-feeds/schedule-price-updates/using-scheduler.mdx

```diff
@@ -1,11 +1,240 @@
+  # batch of updates incurs a minimal gas cost. All fields below are optional (and interpreted as infinity if omitted)
+  # and have the same semantics as the corresponding fields above.
+  early_update:
+    time_difference: 30
+    price_deviation: 0.1
+    confidence_ratio: 0.5
+- ...
+```
+
+Check out [Price feed IDs](https://pyth.network/developers/price-feed-ids) for the complete list of Price feeds supported by Pyth.
+
+### Running the scheduler
+
+The scheduler communicates with [Hermes](https://docs.pyth.network/price-feeds/how-pyth-works/hermes) to get the most recent price updates. 
+Hermes listens to the Pythnet and Wormhole network to get the latest price updates and provides REST and WebSocket APIs for consumers to fetch the updates. 
+Pyth hosts public endpoints for Hermes; however, getting a private endpoint from one of the Hermes RPC providers is recommended for more reliability. 
+Please refer to [this](https://docs.pyth.network/price-feeds/api-instances-and-providers/hermes) document for more information.
+
+
+To run the scheduler, you need to provide the following parameters:
+- `--pyth-contract-address`: The Pyth contract address where the price updates will be pushed. Check out the [Pyth contract addresses](https://docs.pyth.network/price-feeds/contract-addresses) for the complete list of Pyth contracts.
+- `--price-config-file`: Path to the configuration file.
+- `--price-service-endpoint`: Hermes RPC endpoint.
+- `--mnemonic-file`: Path to the file containing the mnemonic for the account that will push the price updates.
+
+
+If you are using the global installation, you can run the scheduler using the following command:
+
+```bash /npx @pythnetwork/price-pusher/ copy
+npx @pythnetwork/price-pusher start -- evm --endpoint wss://example-rpc.com \
+    --pyth-contract-address 0xff1a0f4744e8582DF...... \
+    --price-service-endpoint https://example-hermes-rpc.com \
+    --price-config-file "path/to/price-config.beta.sample.yaml" \
+    --mnemonic-file "path/to/mnemonic.txt" \
+    [--pushing-frequency 10] \
+    [--polling-frequency 5] \
+    [--override-gas-price-multiplier 1.1]
+```
+
+If you are using the cloned repository, you can run the scheduler using the following command:
+
+```bash /npm run/ copy
+npm run start -- evm --endpoint wss://example-rpc.com \
+    --pyth-contract-address 0xff1a0f4744e8582DF...... \
+    --price-service-endpoint https://example-hermes-rpc.com \
+    --price-config-file "path/to/price-config.beta.sample.yaml" \
+    --mnemonic-file "path/to/mnemonic.txt" \
+    [--pushing-frequency 10] \
+    [--polling-frequency 5] \
+    [--override-gas-price-multiplier 1.1]
+```
+
+The scheduler will start monitoring the price feeds and push the updates to the blockchain based on the conditions mentioned in the configuration file.
+
+The scheduler can be used for different chain types. The chain type can be specified as an argument.
```

Comment:
Maybe also say that each chain type has some additional chain-specific options (?)

Do we need to explain those options or do you think they are self-explanatory?

Reply:
We don't need to explain every option in this guide. This guide is already very explanatory and it should be a starting point to start using the Scheduler. If they are really interested they will look at the README and the codebase.